# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../oprofile/gcc-41-anon.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Though I had my own version, taken from upstream.

  - Rene Rebe <rene@exactcode.de>

--- oprofile-0.9.1/libpp/arrange_profiles.cpp.anon	2005-04-13 11:35:24.000000000 -0400
+++ oprofile-0.9.1/libpp/arrange_profiles.cpp	2005-12-05 16:27:42.000000000 -0500
@@ -89,6 +89,8 @@
 	{ "cpu", "specify cpu: or --merge cpu" },
 };
 
+} // anonymous namespace
+
 
 bool profile_classes::matches(profile_classes const & classes)
 {
@@ -129,6 +131,8 @@
 }
 
 
+namespace {
+
 /// We have more than one axis of classification, tell the user.
 void report_error(profile_classes const & classes, axis_types newaxis)
 {
